backend/typescript/migrations/2024.07.08T19.19.43.create-user-pet-activity.ts

```diff
@@ -0,0 +1,68 @@
+import { DataType } from "sequelize-typescript";
+
+import { Migration } from "../umzug";
+
+const TABLE_NAME = "user_pet_activities";
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().createTable(TABLE_NAME, {
+    user_pet_activity_id: {
```

Comment:
i think sequelize automatically creates an id field in the model, so you can just name this 'id', and you can remove the 'user_pet_activity_id' from the model as well

Reply:
You're also missing an id for the pet-behaviours table

Reply:
there was no primary key in the pet behaviours table diagram so I left it out, I can add it if we need it?